modes/magit/README.org

```diff
@@ -0,0 +1,215 @@
+   | evil-next-visual-line       | =j=                      |                          |               |
+   | evil-previous-visual-line   | =k=                      |                          |               |
+   | evil-backward-char          | under =M-x=              |                          | =h=           |
+   | evil-forward-char           | under =M-x=              |                          | =l=           |
+   | evil-search-next            | =n=                      |                          |               |
+   | evil-search-previous        | =N=                      |                          |               |
+   | set-mark-command            | =v= or =V=               | =C-SPC=                  |               |
+   | evil-visual-line            | under =M-x=              | =v= or =V=               |               |
+   | evil-ex                     | =:=                      |                          |               |
+   | evil-search-forward         | =/=                      |                          |               |
+   | evil-scroll-page-up         | =C-b=                    |                          |               |
+   | evil-scroll-down            | =C-d=                    |                          |               |
+   | evil-scroll-page-down       | =C-f=                    |                          |               |
+   | evil-scroll-up              | =C-u= (if =C-u= scrolls) |                          |               |
+   | evil-emacs-state            | =C-z=                    |                          |               |
+   | evil-yank-line              | under =M-x=              | =yy=                     |               |
+   | evil-window-map             | under =M-x=              | =C-w=                    |               |
+   | evil-collection-magit-toggle-text-mode | =C-t/\=                  |                          |               |
+
+
+Any other bindings are meant to be consistent with these.
+
+Use =evil-collection-magit-revert= to revert changes made by evil-collection-magit to the default
+evil+magit behavior.
+
+** To add other common evil commands
+
+Some may want =?= to search backward instead of launching the popup which is
+also bound to =h=. To get this behavior, add the following line after =(require
+'evil-collection-magit)= in your configuration.
+
+#+BEGIN_SRC emacs-lisp
+(evil-define-key evil-collection-magit-state magit-mode-map "?" 'evil-search-backward)
+#+END_SRC
+
+Most (but not all) magit bindings are in =magit-mode-map=, so other commands can
+be bound in this way too.
+
+** To remove commands
+
+Typically, to prevent evil-collection-magit from overriding the default behavior with evil
+and magit loaded, you should bind the respective key to =nil= after loading
+evil-collection-magit. For example, to make =escape= behave as default
+
+#+BEGIN_SRC emacs-lisp
+(evil-define-key* evil-collection-magit-state magit-mode-map [escape] nil)
+#+END_SRC
+
+* Known Conflicts
```

Comment:
No titlecase for consistency with other sections.

Reply:
Should be consistent already.

Reply:
I've looked again, and actually many more sections than those 2 are titlecased (or partially).  But many are sentence-cased.